lambda/storeHistoricalDataInDB.ts

```diff
@@ -0,0 +1,48 @@
+import {
+	type TimestreamWriteClient,
+	type _Record,
+	WriteRecordsCommand,
+	RejectedRecordsException,
+} from '@aws-sdk/client-timestream-write'
+
+export const storeHistoricalDataInDB =
+	(tsw: TimestreamWriteClient, dbName: string, tableName: string) =>
+	async (records: _Record[]): Promise<void> => {
+		const recordsToTimestream = []
+		//It is only possible to send 100 records at a time
+		if (records.length > 100) {
+			let j = 0
+			for (let i = 99; j < records.length; i += 100) {
+				recordsToTimestream.push(records.slice(j, i))
+				j = i
+			}
+		} else {
```

Comment:
This looks strange, and has no test.
Instead, extract this into a separate generic function that can chunk arrays.